src/Sylius/Bundle/AdminBundle/Resources/private/js/app.js

```diff
@@ -26,6 +26,8 @@ import './sylius-notification';
 import './sylius-product-images-preview';
 import './sylius-product-slug';
 import './sylius-taxon-slug';
+import './sylius-chart';
```

Comment:
What is this?

Reply:
Idk.. this was in the first merged PR

Reply:
May you try to remove it and check if it is still needed?

Reply:
I've remove it, I should try it again locally